FreeRTOS/Demo/Posix_GCC/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.15)
+
+project( posix_demo )
+
+set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_WINDOWS_")
+
+if( DEFINED TRACE_ON_ENTER AND TRACE_ON_ENTER )
```

Comment:
Is there a reason all of these if checks have the variable twice?

Reply:
Thanks for the catch. The first defined check is redundant. Update in the implementation.